src/block/VanillaBlocks.php

```diff
@@ -1249,7 +1264,13 @@ private static function registerWoodenBlocks() : void{
 			$idName = fn(string $suffix) => $woodType->name() . "_" . $suffix;
 
 			self::register($idName(mb_strtolower($woodType->getStandardLogSuffix() ?? "log", 'US-ASCII')), new Wood(WoodLikeBlockIdHelper::getLogIdentifier($woodType), $name . " " . ($woodType->getStandardLogSuffix() ?? "Log"), $logBreakInfo, $woodType));
-			self::register($idName(mb_strtolower($woodType->getAllSidedLogSuffix() ?? "wood", 'US-ASCII')), new Wood(WoodLikeBlockIdHelper::getAllSidedLogIdentifier($woodType), $name . " " . ($woodType->getAllSidedLogSuffix() ?? "Wood"), $logBreakInfo, $woodType));
+			if(WoodType::BAMBOO()->equals($woodType)) {
+				self::register("bamboo_mosaic", new Planks(new BID(Ids::BAMBOO_MOSAIC), "Bamboo Mosaic", $planksBreakInfo, $woodType));
+				self::register("bamboo_mosaic_slab", new WoodenSlab(new BID(Ids::BAMBOO_MOSAIC_SLAB), "Bamboo Slab", $planksBreakInfo, $woodType));
+				self::register("bamboo_mosaic_stairs", new WoodenStairs(new BID(Ids::BAMBOO_MOSAIC_STAIRS), "Bamboo Stairs", $planksBreakInfo, $woodType));
```

Comment:
I think it's misleading to pass the wood type here. We already have variants of these blocks which will have a BAMBOO type.

Reply:
Add a new WoodType::BAMBOO_MOSAIC() doesn't make sense to me, I expand it to VanillaBlocks::registerBambooMosaicBlocks() as a possible solution

Reply:
I'm not convinced bamboo mosaic should be considered a wood type at all.